CONTRIBUTORS.md

```diff
@@ -22,3 +22,5 @@
   Documentation and general feedback
 - Rim Rihana (Merck KGaA, Darmstadt, Germany):
   Human readable output for search spaces
+- Di Jin (Merck Life Science KGaA, Darmstadt, Germany)
+  Continuous cardinality constraint
```

Comment:
maybe write `cardinality constraints` because then this line will also include the planned future contributions

Reply:
can change and will change. But you do remember that I had it exactly like that before (for the exact same reason) and it was you who asked me to change it, right? 😄

Reply:
changed it (back)

Reply:
well I blame the fact that I didnt anticipate the implicit plans you have for the next PRs :) but sry